src/multivariate/mvnormal.jl

```diff
@@ -80,6 +80,27 @@ abstract type AbstractMvNormal <: ContinuousMultivariateDistribution end
 insupport(d::AbstractMvNormal, x::AbstractVector) =
     length(d) == length(x) && all(isfinite, x)
 
+"""
+    minimum(d::AbstractMvNormal)
+
+Return the minimum of the support of each dimension of `d`.
+"""
```

Comment:
The docstrings seem a bit redundant, maybe it would be sufficient to have one docstring for `minimum(d::Distribution)` etc.?

Reply:
Where, as in which file, should I define this?

Reply:
Docstring of `minimum(::Distribution)` should be just "Return the `minimum` of the support of `d`."

Reply:
I.e. just widen the signature of 
```
  minimum(d::UnivariateDistribution)

  Return the minimum of the support of d.
```

Reply:
Do I need an implementation for minimum(::Distribution)? Or can just a docstrring be added?